services/app-graphql/src/schema.graphql

```diff
@@ -4,36 +4,139 @@ schema {
 }
 
 type Query {
+    """
+    fetchCurrentUser returns user information for the currently logged in User
+
+    If no user is currently logged in, the http request will return a 403 error, no graphQL body will be returned.
+    """
     fetchCurrentUser: User!
+
+    """
+    fetchHealthPlanPackage returns a specific HealthPlanPackage by id
+
+    If a package with the given ID cannot be found, this query returns undefined
+    CMS users cannot fetch a DRAFT HealthPlanPackage
+
+    Errors:
+    - ForbiddenError:
+        - A state user from a different state called this.
+        - A CMSUser attempted to fetch a DRAFT HealthPlanPackage
+    """
     fetchHealthPlanPackage(
         input: FetchHealthPlanPackageInput!
     ): FetchHealthPlanPackagePayload!
+
+    """
+    indexHealthPlanPackages returns all of the HealthPlanPackages the current user can see.
+
+    StateUsers can find all the packages for their state
+    CMSUsers can find all the packages that do not have the DRAFT status
+    """
     indexHealthPlanPackages: IndexHealthPlanPackagesPayload!
 }
 
 type Mutation {
+    """
+    createHealthPlanPackage creates a new HealthPlanPackage.
+
+    The new HealthPlanPackage is created with a single revision and with
+    the DRAFT status. The stateCode of the package will be set to the
+    state the user that calls this is from.
+
+    This can only be called by StateUsers
+
+    Errors:
+    - ForbiddenError: If a non-state users calls this mutation
+    - UserInputError: ProgramID not found in this state's programs
+    """
     createHealthPlanPackage(
         input: CreateHealthPlanPackageInput!
     ): CreateHealthPlanPackagePayload!
+
+    """
+    updateHealthPlanFormData updates a single package with the current
+    state of the form encoded as a protobuf.
+
+    The package must be either in the DRAFT or UNLOCKED state.
+    Only a state user from the state this package is attached to can call this mutation
+
+    There are some fields in the healthPlanFormData type that must not be modified
+    by this mutation. They are set on the initial submission and are only changed by the server:
+    - id
+    - stateCode
+    - stateNumber
+    - createdAt
+    - updatedAt
+
+    Errors:
+    - ForbiddenError:
+        - A non-state user called this
+        - A state user from a different state called this.
```

Comment:
What can a CMS user do, or not?

Reply:
You mention it in a previous paragraph; just wondering if we want to be comprehensive in the Errors subsection.

Reply:
I’ll change non-state user to “CMSUser” since that’s the only other type of user we have right now and is more clear, does that cover it?